test_ffi/tests/test.js

```diff
@@ -388,13 +400,13 @@ dylib.symbols.call_stored_function_2(20);
 const nestedCallback = new Deno.UnsafeCallback(
   { parameters: [], result: "void" },
   () => {
-    dylib.symbols.call_stored_function_2(10);
+    symbols.call_stored_function_2(10);
   },
 );
-dylib.symbols.store_function(ptr(nestedCallback));
+symbols.store_function(ptr(nestedCallback));
 
-dylib.symbols.store_function(null);
-dylib.symbols.store_function_2(null);
+symbols.store_function(null);
```

Comment:
Why the `dylib.` removals? Does this allow the other calls to be optimised as well? If not, then maybe revert this change as unnecessary.

Reply:
If unnecessary, I agree. This is a complex patch and should be kept as minimal as possible.

Reply:
Reverted